src/metabase/search/appdb/index.clj

```diff
@@ -40,7 +40,9 @@
 
 (defonce ^:dynamic ^:private *indexes* (atom {:active nil, :pending nil}))
 
-(def ^:private ^:dynamic *mocking-tables* false)
+(def ^:dynamic *temp-index-tables*
```

Comment:
Not great leaking this dynamic var outside the namespace, and both places this is don't seem like good ideas to me, at first glance.